tap_pardot/client.py

```diff
@@ -85,7 +88,13 @@ def _make_request(self, method, url, params=None):
             url,
             params,
         )
-        response = requests.request(
+
+        request_session = requests.Session()
+        retries = Retry(total=100, backoff_factor=1, status_forcelist=[502, 503, 504])
```

Comment:
100 total retries.. Is that global/total over the lifetime of the session, or does it get reset every time it succeeds? And if that is the case, 100 might be a bit too much :D